boards/common/stm32/include/f2f4f7/cfg_clock_default_168.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *               2017 OTA keys S.A.
+ *               2018-2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_stm32
+ * @{
+ *
+ * @file
+ * @brief       Default STM32F4 clock configuration for 168MHz boards
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef F2F4F7_CFG_CLOCK_DEFAULT_168_H
+#define F2F4F7_CFG_CLOCK_DEFAULT_168_H
+
+#include "f2f4f7/cfg_clock_common.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Clock PLL settings (180MHz)
+ *
+ * The PLL settings provided here can be used for USB on CPU with a max
+ * frequency of 180MHz.
+ * @{
+ */
+/* The following parameters configure a 168MHz system clock with HSE
+   (8MHz, 12MHz or 16MHz) or HSI (16MHz) as PLL input clock */
```

Comment:
Why 168Mhz?

Reply:
It's because, in RIOT, we use 168MHz instead of 180MHz for compatibility with USB 48MHz required clock: it's not possible to configure a 48MHz on PLLQ when PLLP is configured for 180MHz (no common divisor).
There's the same issue with M4 that can have a 100MHz max core clock, in RIOT, they are configured to 96MHz.

See #15043 which improves the situation: use 100/180MHz by default, except when `periph_usbdev` is used. This PR also removes the useless `cfg_clock_default_96.h` and `cfg_clock_default_168.h` default clock configuration headers.

Reply:
ok